keyless/pepper/service/src/main.rs

```diff
@@ -37,6 +40,14 @@ async fn handle_request(req: Request<Body>) -> Result<Response<Body>, Infallible
         (&Method::GET, "/about") => {
             build_response(origin, StatusCode::OK, ABOUT_JSON.deref().clone())
         },
+        (&Method::GET, "/cached/keyless-config") => build_response_for_optional_resource(
```

Comment:
why do you call this `/cached` instead of `v0/` or `v1/` or whatever?

Reply:
it's not API version specific so better not have `vx` in the path?